test/controllers/sessions_controller_test.rb

```diff
@@ -38,7 +38,6 @@ class SessionsControllerTest < ActionController::TestCase
     test "#new should render a full-page if the shop param doesn't exist" do
       get :new
       assert_response :ok
-      assert_template :new
```

Comment:
In Rails 5.0, `assert_template` only exists in https://github.com/rails/rails-controller-testing now. Figured it's easier to remove the need for it since the test is still useful.